src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/LeftCurlyCheck.java

```diff
@@ -162,6 +165,7 @@ public void setIgnoreEnums(boolean ignoreEnums) {
         return CommonUtils.EMPTY_INT_ARRAY;
     }
 
+    // -@cs[JavaNCSS] Suppress NCSS, because of the cases
```

Comment:
Instead of turning off the check, you need to rewrite the switch.
Turn the biggest block with many LITERAL_ * cases to "default:" and "default:" to simple "case LITERAL_SWITCH:".
In addition, you need to correct the comment near the "default:" because it is outdated and erroneous.

Reply:
to avoid violation of JavaNCSS, you need to make some case blocks like:
```
startToken = xxxxxxxxxxxx(wwww);
brace = xxxxxxxxxx(zzzz);
```